Classes/View Controllers/TabBarController.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  TabBarController.swift
+//  Freetime
+//
+//  Created by Rizwan on 26/02/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+final class TabBarController: UITabBarController {
+    enum Tab: Int {
+        case inbox = 0
```

Comment:
An enum rawValue of `Int` will default to `0` and increment by `1` for every case, unless specified